admin/cluster_setup.py

```diff
@@ -11,23 +11,159 @@
 from pipes import quote as shell_quote
 
 from eliot import FileDestination, add_destination, write_failure
+from pyrsistent import pvector
+from txeffect import perform
 
-from twisted.internet.defer import inlineCallbacks
-from twisted.python.usage import UsageError
+from twisted.internet.defer import DeferredList, inlineCallbacks
 from twisted.python.filepath import FilePath
+from twisted.python.usage import UsageError
 
 from .acceptance import (
+    CLOUD_PROVIDERS,
     ClusterIdentity,
     CommonOptions,
+    LibcloudRunner as OldLibcloudRunner,
```

Comment:
Well, the "new" `LibcloudRunner` is designed as a drop-in replacement for the old `LibcloudRunner` and the replacement is supposed to happen "very soon". Hence, the pattern.  But I am open to changing the name.